fuzzing/build.gradle.kts

```diff
@@ -13,5 +13,7 @@ dependencies {
 
 tasks.register<Copy>("copyToFuzzOut") {
```

Comment:
The task should be `Sync` rather than `Copy` to remove stale files if some of the classes are removed from build to build